### PR TITLE
BC-7510 - Add Training Link to Sidebar

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Short Description

The sidebar menu item "Trainings" was taken out and needs to be added back.

Includes fixes for the old layout:

- setting correct top position for content in old layout
- fixing cloud logo in sidebar by re-adding old logo files

## Links to Ticket and related Pull-Requests

BC-7510

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

**Review by MartinSchuhmacher — DISMISSED** [review]

File: src/components/templates/DefaultWireframe.unit.ts [hunk 1]
```diff
@@ -73,24 +86,19 @@ describe("DefaultWireframe", () => {
 	});
 
 	it("displays content in slot", () => {
-		const wrapper = mount(DefaultWireframe, {
-			global: {
-				plugins: [createTestingVuetify(), createTestingI18n()],
-			},
-			props: { headline: "dummy title", fullWidth: false },
+		const wrapper = setup({
+			props: { fullWidth: true, headline: "dummy title" },
```
Comment (virgilchiriac) [inline src/components/templates/DefaultWireframe.unit.ts:90]: not sure if on purpose, but why fullwidth was changed from false to true?

File: src/components/templates/RoomWrapper.vue [hunk 2]
```diff
@@ -3,6 +3,7 @@
 		ref="main"
 		:full-width="false"
 		:fab-items="fabItems"
+		:env-config-module="envConfigModule"
```
Comment (virgilchiriac) [inline src/components/templates/RoomWrapper.vue:6]: why this change was needed?

File: src/components/templates/DefaultWireframe.unit.ts [hunk 3]
```diff
@@ -60,11 +76,8 @@ describe("DefaultWireframe", () => {
 	});
 
 	it("has small width", () => {
-		const wrapper = mount(DefaultWireframe, {
-			global: {
-				plugins: [createTestingVuetify(), createTestingI18n()],
-			},
-			props: { headline: "dummy title", fullWidth: false },
+		const wrapper = setup({
+			props: { headline: "dummy title" },
```
[the remark this answers is not in the document]
Comment (odalys-dataport) [reply src/components/templates/DefaultWireframe.unit.ts:80]: It's not a required prop anymore

File: src/components/templates/RoomWrapper.vue (hunk 2 above)
Comment (odalys-dataport) [reply src/components/templates/RoomWrapper.vue:6]: It's a hacky solution. I added a TODO comment to remove this when we have gotten rid of the old layout. 

When I injected the envConfigModule into DefaultWireframe *every* page test was failing due to the unresolved injection as the DefaultWireframe is used in all of them.

File: src/components/templates/DefaultWireframe.unit.ts (hunk 1 above)
Comment (odalys-dataport) [reply src/components/templates/DefaultWireframe.unit.ts:90]: It's from copy pasting, yes. But the value of the prop `fullWidth` doesn't matter in this test.

**Review by virgilchiriac — APPROVED** [review]
